### PR TITLE
Combine multiple hardforks into one

### PR DESCRIPTION
**Pull request by VAIBHAVJINDAL3012**
# Description
This PR merges all the following hf
1.GetValidatorSetRotationStopHeight
2.GetMilestoneHardForkHeight()
3.GetNoAckProposerCheckHeight()
into the one single aalborg hf

### CODE REVIEW

File: checkpoint/side_milestone_handler.go [hunk 1]
```diff
@@ -22,12 +22,6 @@ func SideHandleMsgMilestone(ctx sdk.Context, k Keeper, msg types.MsgMilestone, c
 	// logger
 	logger := k.Logger(ctx)
 
-	//Check whether the chain has reached the hard fork length
-	if ctx.BlockHeight() < helper.GetMilestoneHardForkHeight() {
```
Comment (temaniarpit27) [inline checkpoint/side_milestone_handler.go:26]: why have we removed this?

File: checkpoint/side_milestone_handler_test.go [hunk 2]
```diff
@@ -33,27 +33,6 @@ func (suite *SideHandlerTestSuite) TestSideHandleMsgMilestone() {
 
 	borChainId := "1234"
 
-	suite.Run("Failure-Before Hard Fork", func() {
```
Comment (temaniarpit27) [inline checkpoint/side_milestone_handler_test.go:36]: why removed this?

File: checkpoint/side_milestone_handler.go (hunk 1 above)
Comment (VAIBHAVJINDAL3012) [reply checkpoint/side_milestone_handler.go:26]: This is redundant, Msg will never reach this if the chain has not crossed the hardfork.

File: checkpoint/side_milestone_handler_test.go (hunk 2 above)
Comment (VAIBHAVJINDAL3012) [reply checkpoint/side_milestone_handler_test.go:36]: This was related to HF check in sidehandler.

**Review by temaniarpit27 — APPROVED** [review]

**Review by manav2401 — APPROVED** [review]

**Pull request merged**